test/cases/aws.sh

```diff
@@ -234,14 +233,23 @@ if [ "$ARCH" == "aarch64" ]; then
     sed -i s/t3.medium/a1.large/ "${TEMPDIR}/resource-file.json"
 fi
 
+# The weird sintax with the CIV_OPTION variables is due to problems with bash quoting and
+# having options with whitespaces in them. Please do not spend more hours trying to fix it
 sudo "${CONTAINER_RUNTIME}" run \
     -a stdout -a stderr \
     -e AWS_ACCESS_KEY_ID="${V2_AWS_ACCESS_KEY_ID}" \
     -e AWS_SECRET_ACCESS_KEY="${V2_AWS_SECRET_ACCESS_KEY}" \
     -e AWS_REGION="${AWS_REGION}" \
     -v "${TEMPDIR}":/tmp:Z \
     "${CONTAINER_CLOUD_IMAGE_VAL}" \
-    python cloud-image-val.py "${CIV_OPTIONS[@]}" && RESULTS=1 || RESULTS=0
+    python cloud-image-val.py \
+    ${CIV_OPTION_R:+"$CIV_OPTION_R"} \
+    ${CIV_OPTION_O:+"$CIV_OPTION_O"} \
+    ${CIV_OPTION_T:+"$CIV_OPTION_T"} \
+    ${CIV_OPTION_M:+"$CIV_OPTION_M"} \
+    ${CIV_OPTION_D:+"$CIV_OPTION_D"} \
+    ${CIV_OPTION_S:+"$CIV_OPTION_S"} \
```

Comment:
`CIV_OPTION_T` and `CIV_OPTION_S` don't seem to be set anywhere. Is this intentional?

Reply:
It is, we might want to set this variables when the script is being executed from CIV ci